dev/dune

```diff
@@ -12,6 +12,8 @@
   (deps dune-dbg.in
         ../checker/coqchk.bc
         ../topbin/coqc_bin.bc
+        ../ide/coqide_main.bc
+        ../topbin/coqtop_byte_bin.bc
```

Comment:
Why coqtop?

Reply:
Oh, this likely sneaked in from a rebase as this PR dates back from the times of `coqtop` based debug.

Reply:
>Oh, this likely sneaked in from a rebase as this PR dates back from the times of coqtop based debug.

Please remove then.